src/cmd/lint.js

```diff
@@ -51,6 +51,7 @@ export type LintCmdParams = {|
   warningsAsErrors?: boolean,
   ignoreFiles?: Array<string>,
   artifactsDir?: string,
+  runAsBinary?: boolean,
```

Comment:
This should move to a function option (which are [passed here](https://github.com/ccarruitero/web-ext/blob/a0a2bfcd3667c868ae729f9a39b83791d263d5f5/src/cmd/lint.js#L57)) rather than a command line parameter.